package.json

```diff
@@ -1,69 +1,59 @@
 {
-  "name": "svelteapp",
+  "name": "dp-census-maps",
```

Comment:
I know this is confusing, but I think the apps name might be best as dp-census-atlas, unless we move everything again?

Reply:
Agreed. (This name isn't used for anything, as it's not a published packed but yeah, let's make it consistent.)